packages/location-management/src/components/LocationUnitAddEdit/tests/index.test.tsx

```diff
@@ -0,0 +1,256 @@
+import flushPromises from 'flush-promises';
+import { mount } from 'enzyme';
+import React from 'react';
+import { history } from '@onaio/connected-reducer-registry';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route, Router } from 'react-router';
+import fetch from 'jest-fetch-mock';
+import { store } from '@opensrp/store';
+
+import { id, locationUnitgroup, sampleHierarchiesList, sampleLocationUnit } from './fixtures';
+import LocationUnitAddEdit from '..';
+import { act } from 'react-dom/test-utils';
+import { sampleHierarchy } from '../../LocationUnitView/tests/fixtures';
+
+describe('location-management/src/components/LocationUnitAddEdit', () => {
+  it('renders without crashing', async () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <LocationUnitAddEdit />
+        </Router>
+      </Provider>
+    );
+    expect(wrapper.props()).toMatchSnapshot();
+
+    await act(async () => {
+      await flushPromises();
+    });
+    wrapper.unmount();
+  });
+
+  it('renders without crashing when editting', async () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={[{ pathname: `/${id}`, hash: '', search: '', state: {} }]}>
+          <Route path={'/:id'} component={LocationUnitAddEdit} />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.props()).toMatchSnapshot();
```

Comment:
I also feel we can do our with this snapshot

Reply:
i update whole tests for this file